source/community.applications/usr/local/emhttp/plugins/community.applications/scripts/languageInstall.sh

```
@@ -1,23 +1,5 @@
-#   $2 - Language pack identifier (e.g. `de_DE`).
-# Returns:
-#   Exit status of the final `language` invocation; writes that command's
-#   progress output to stdout.
-# ---------------------------------------------------------------------------
-if [ -z "${1:-}" ] || [ -z "${2:-}" ]; then
-	echo "Usage: $0 <install|update|remove> <language-pack-id>" >&2
-	exit 2
-fi
 if [ "$1" = "update" ]; then
-	/usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/language check "$2"
+	/usr/local/emhttp/plugins/dynamix.plugin.manager/scripts/language check $2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)